### PR TITLE
#5 Spelling mistake fixes

### PR DESCRIPTION
**Pull request by JakeyD11**

### CODE REVIEW

**Review by carys23 — APPROVED** [review]

File: JakeyD11/index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h2>My Favourite Music</h2>
   <tr>
     <td>ACDC</td>
     <td>Back in Black</td>
-    <td>180</td>
+    <td>1980</td>
   </tr>
 </table>
```
Comment (tamjankowska) [inline JakeyD11/index.html:50]: Shall we get rid of some of this white space below?

File: JakeyD11/style.css [hunk 2]
```diff
@@ -56,4 +56,4 @@ th, td {
     color: white;
 }
 .site-header nav a:hover {
-     color: pink;
\ No newline at end of file
+     color: pink;}
```
Comment (tamjankowska) [inline JakeyD11/style.css:59]: Maybe pop your closing curly on a new line?

**Review by tamjankowska — CHANGES_REQUESTED** [review]

File: JakeyD11/index.html (hunk 1 above)
Comment (JakeyD11) [reply JakeyD11/index.html:50]: done

File: JakeyD11/style.css (hunk 2 above)
Comment (JakeyD11) [reply JakeyD11/style.css]: done :)

**Review by jadavey91 — DISMISSED** [review]
Cool

**Review by boothscript — CHANGES_REQUESTED** [review]
you have two closing html tags

**Review by Edwine-R — APPROVED** [review]

**Comment by JakeyD11** [comment]
> you have two closing html tags

done thanks

**Review by boothscript — APPROVED** [review]
Looks good

**Review by jthomasw88 — APPROVED** [review]
looks ok to me matey

**Review by Edwine-R — APPROVED** [review]

**Pull request merged**